### DISCUSSION TITLE
Scale

### DISCUSSION THREAD

**Discussion by Timerlen** [mozman/ezdxf#370] [Q&A]
Good morning 
I have a question about the scale in dxf files. 
![1](https://user-images.githubusercontent.com/80266841/112294674-5fc8ac00-8c93-11eb-8cd2-5f6e6c6fc73e.jpg)
create a dxf file. It is a circle with a diameter of 1000 mm. In order for the whole drawing to fit in the frame, the program automatically gives it a scale, but the dimensions are the same as in the original and this is indicated in the scale table in the lower right corner 
![2](https://user-images.githubusercontent.com/80266841/112294755-753dd600-8c93-11eb-9f2e-fbcf02a53d10.jpg)
after exporting the file to dxf, the scale in the table is 1: 1 the dimensions are original and the actual dimensions are reduced by the scale. 
![3](https://user-images.githubusercontent.com/80266841/112295628-5ee44a00-8c94-11eb-9a53-244b0150d6ee.jpg)
My question is whether in the dxf file there is information about what value was used for scaling?

**Comment by mozman** [comment]
Hi,

this is not a support forum for general CAD related questions. Please use the help or support forum of your CAD application.

**Reply by Timerlen** [reply]
Ok thank you for the answer. I asked out of curiosity because I see that you are an expert in the field of dxf. Searching the forums I saw you have a lot of activity about dxf. There is a high probability that information about the scale disappears with the export to dxf
